udp/message/getmid.go

```diff
@@ -15,6 +16,9 @@ func GetMID() uint16 {
 
 func RandMID() uint16 {
 	b := make([]byte, 4)
-	rand.Read(b)
+	_, err := rand.Read(b)
+	if err != nil {
+		log.Fatalf("cannot get random value: %v", err)
```

Comment:
Is there anything else we can do here? Aborting within a library should be avoided, but I couldn't think of anything else. It could return error, but the error would just move to

```go
var msgID = uint32(RandMID())
```
.

Reply:
I think we can use the time when `rand.Read(b)` fails. It's just mid which is used for pairing messages. And this func i think is used only for initialization.,